packages/core/src/Link/Link.tsx

```diff
@@ -43,23 +44,38 @@ export interface ILinkProps
     SpaceProps,
     React.HTMLAttributes<HTMLAnchorElement>,
     React.RefAttributes<unknown> {
+  children?: React.ReactNode | string
   color?: string
+  disabled?: boolean
```

Comment:
I'm wondering if this should be closer to a conventional boolean name like `isDisabled`? If this is already consistent with how other components name it in `core` though that's fine.

Reply:
I was considering that too but my reasoning was 1. like you said all our components already use `disabled` and 2. that's the attribute name in plain html for things that are disableable.